worker/uniter/uniter.go

```diff
@@ -269,16 +269,21 @@ func (u *Uniter) loop(unitTag names.UnitTag) (err error) {
 	defer func() {
 		// If this is a CAAS unit, then dead errors are fairly normal ways to exit
 		// the uniter main loop, but the parent operator agent needs to keep running.
+		errorString := "<unknown>"
+		if err != nil {
+			errorString = err.Error()
+		}
 		if errors.Cause(err) == ErrCAASUnitDead {
```

Comment:
We could change `ErrCAASUnitDead` error from "unit dead" to "caas unit dead" and set `errorString = ErrCAASUnitDead.String()`